src/webgpu/shader/execution/expression/call/builtin/texture_utils.ts

```diff
@@ -1308,7 +1308,7 @@ export async function checkCallResults<T extends Dimensionality>(
       ? getMaxFractionalDiffForTextureFormat(texture.descriptor.format)
       : 0;
 
-  for (let callIdx = 0; callIdx < calls.length /*&& errs.length === 0*/; callIdx++) {
+  for (let callIdx = 0; callIdx < calls.length && errs.length === 0; callIdx++) {
```

Comment:
nit: why not `break` at end of `if (bad) {}` branch?

Reply:
done